### PR TITLE
Add option for tech window zoom level

### PR DESCRIPTION
**Pull request by dbenage-cx**
Adds config option for saving the zoom level of tech layout window

Partial work towards #529

### CODE REVIEW

File: UI/OptionsWnd.cpp [hunk 1]
```diff
@@ -491,6 +491,7 @@ OptionsWnd::OptionsWnd():
     CreateSectionHeader(current_page, 0, UserString("OPTIONS_TECH_SPACING"));
     DoubleOption(current_page, 0, "UI.tech-layout-horz-spacing", UserString("OPTIONS_HORIZONTAL"));
     DoubleOption(current_page, 0, "UI.tech-layout-vert-spacing", UserString("OPTIONS_VERTICAL"));
+    DoubleOption(current_page, 0, "UI.tech-layout-zoom-scale", UserString("OPTIONS_ZOOM"));
```
Comment (geoffthemedio) [inline UI/OptionsWnd.cpp]: align UserString

File: UI/TechTreeWnd.cpp [hunk 2]
```diff
@@ -37,6 +37,7 @@ namespace {
     void AddOptions(OptionsDB& db) {
         db.Add("UI.tech-layout-horz-spacing", UserStringNop("OPTIONS_DB_UI_TECH_LAYOUT_HORZ_SPACING"), 1.0,  RangedStepValidator<double>(0.25, 0.25, 4.0));
         db.Add("UI.tech-layout-vert-spacing", UserStringNop("OPTIONS_DB_UI_TECH_LAYOUT_VERT_SPACING"), 0.75, RangedStepValidator<double>(0.25, 0.25, 4.0));
+        db.Add("UI.tech-layout-zoom-scale", UserStringNop("OPTIONS_DB_UI_TECH_LAYOUT_ZOOM_SCALE"), 1.0, RangedStepValidator<double>(1.0, -25.0, 10.0));
```
Comment (geoffthemedio) [inline UI/TechTreeWnd.cpp]: Align UserStringNop

File: UI/TechTreeWnd.cpp [hunk 3]
```diff
@@ -1117,6 +1118,7 @@ void TechTreeWnd::LayoutPanel::SetScale(double scale) {
         scale = MAX_SCALE;
     if (m_scale != scale)
```
Comment (geoffthemedio) [inline UI/TechTreeWnd.cpp]: not your patch, but this if seems pretty unnecessary...

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -1370,6 +1370,9 @@ The horizontal spacing to be placed between techs in the tech screen, in multipl
 OPTIONS_DB_UI_TECH_LAYOUT_VERT_SPACING
 The vertical spacing to be placed between techs in the tech screen, in multiples of the height of a single theory tech.
 
+OPTIONS_DB_UI_TECH_LAYOUT_ZOOM_SCALE
+Adjusts the scale to zoom the tech window, from -25 to 10.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: range in description unnecessary and meaningless to the user

**Comment by dbenage-cx** [comment]
rebased

**Comment by dbenage-cx** [comment]
Note [commented out](https://github.com/freeorion/freeorion/pull/561/files#diff-ebebeb87a5c1fe446a3b437621288016L1047) fps option assignments, as they are unused.  Left comments in place as they are more recent (9/11/15)

**Pull request merged**